src/components/PreferencesModal/PreferencesModal.tsx

```diff
@@ -32,25 +32,26 @@ const Container = styled.div`
 const Header = styled.h1`
   margin: 0;
   padding: 1em 0;
+  font-weight: bold;
 `
 
 const TabNav = styled.nav`
   width: 200px;
-  margin-left: 30px;
```

Comment:
Using percentage is not a great idea. We should set max-width and place content center.